src/Polly/AsyncPolicy.GenericImplementation.cs

```diff
@@ -13,7 +13,7 @@ public abstract partial class AsyncPolicy<TResult>
     protected abstract Task<TResult> ImplementationAsync(
         Func<Context, CancellationToken, Task<TResult>> action,
         Context context,
-        CancellationToken cancellationToken,
-        bool continueOnCapturedContext
+        bool continueOnCapturedContext,
```

Comment:
Just scan reading this PR before a proper review tomorrow, the public API surface area like this **must not** be changed to resolve these warnings. Anything this isn't internal or private must be suppressed only.

Changing the order of the parameters in the public API surface is a breaking change we cannot accept.